modules/commons/src/main/frontend/webpack.config.js

```diff
@@ -1,54 +1,4 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-const { CleanWebpackPlugin } = require('clean-webpack-plugin');
-const WebpackAssetsManifest = require('webpack-assets-manifest');
-
-module_name = "cards-commons";
-
 module.exports = {
-  mode: 'development',
-  entry: {
-    [module_name + '.404']: './src/components/404.js',
-    [module_name + '.GenericErrorPage']: './src/components/GenericErrorPage.js',
-  },
-  plugins: [
-    new CleanWebpackPlugin(),
-    new WebpackAssetsManifest({
-      output: "assets.json"
-    })
-  ],
-  module: {
-    rules: [
-      {
-        test: /\.(js|jsx)$/,
-        exclude: /node_modules/,
-        use: ['babel-loader']
-      }
-    ]
-  },
-  resolve: {
-    extensions: ['*', '.js', '.jsx']
-  },
-  output: {
-    path: __dirname + '/dist/SLING-INF/content/libs/cards/resources/',
-    publicPath: '/',
-    filename: '[name].[contenthash].js'
-  }
-};
\ No newline at end of file
+  ['cards-commons.404']: './src/components/404.js',
+  ['cards-commons.GenericErrorPage']: './src/components/GenericErrorPage.js',
```

Comment:
```suggestion
['cards-commons.404']: './src/components/404.js'
['cards-commons.GenericErrorPage']: './src/components/GenericErrorPage.js'
```

No indentation, and the trailing comma is removed, always to be added by the python script.